english_to_morse_code.rb

```diff
@@ -0,0 +1,30 @@
+def decode_char(alphabet)
+  morse = {
+    '.-' => 'A', '-...' => 'B', '-.-.' => 'C', '-..' => 'D', '.' => 'E', '..-.' => 'F',
+    '--.' => 'G', '....' => 'H', '..' => 'I', '.---' => 'J', '-.-' => 'K', '.-..' => 'L',
+    '--' => 'M', '-.' => 'N', '---' => 'O', '.--.' => 'P', '--.-' => 'Q', '.-.' => 'R',
+    '...' => 'S', '-' => 'T', '..-' => 'U', '...-' => 'V', '.--' => 'W', '-..-' => 'X',
+    '-.--' => 'Y', '--..' => 'Z'
+  }
+  morse[alphabet]
+end
+
+def decode_word(alphabet)
+  word_split = alphabet.split
+  decoded_word = ''
+  word_split.each do |letter|
+    decoded_word += decode_char(letter)
+  end
+  decoded_word
+end
+
+def decode(str)
+  sentence_split = str.split
+  full_sentence_decoded = ' '
+  sentence_split.each do |word|
+    full_sentence_decoded += decode_word(word).to_s
+  end
+  full_sentence_decoded.strip
+end
```

Comment:
- [ ] The output from these methods is not the expected one. You are missing spaces between your words in the decoded result. Let's fix that

* Expected
```
"A"
"MY"
"MY NAME"
"A BOX FULL OF RUBIES"
```
* Your results
![Screenshot from 2021-12-16 19-19-46](https://user-images.githubusercontent.com/40503704/146427533-57b2b606-bce5-4be8-88e0-277e126cd376.png)